op-node/node/node.go

```diff
@@ -537,10 +610,25 @@ func (n *OpNode) Close() error {
 		n.closed.Store(true)
 	}
 
+	if n.halted {
```

Comment:
The code changes are fine, I'm a little curious about how this halt behavior interacts with different types of users. I'm a little worried that with config/setup errors we're suppressing it for too long for typical users here.

Reply:
It's only idling this way if it's an intentional halt and not when it's an actual op-node error, so it won't delay any actual error from being surfaced to the user.